config/application.rb

```diff
@@ -8,14 +8,24 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
+class JsonLogSerializer < ActiveSupport::Logger::SimpleFormatter
+  def call(severity, timestamp, _progname, message)
+    {
+      type: severity,
```

Comment:
```suggestion
      level: severity,
```
cf la [documentation Datadog](https://docs.datadoghq.com/fr/logs/log_configuration/pipelines/?tab=status#attribut-status), le status du log est récupéré par défaut dans une clé `status` ou `level`, autant rester dans les normes de ce qu'il se fait